src/main/resources/templates/email/feedbackToInterviewee.txt

```diff
@@ -0,0 +1,26 @@
+Thank you for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewer has submitted feedback for your interview which you can view below. 
```

Comment:
Grammar nit: there should be a comma before "which"

Reply:
Done